src/jdk.jartool/share/classes/sun/tools/jar/resources/jar.properties

```diff
@@ -143,6 +143,16 @@ warn.validator.concealed.public.class=\
         Warning: entry {0} is a public class\n\
         in a concealed package, placing this jar on the class path will result\n\
         in incompatible public interfaces
+warn.validator.duplicate.cen.entry=\
+        Warning: {0} copies of {1} is detected in central directory
```

Comment:
I would change:

`Warning: {0} copies of {1} is detected in central directory`
to:
`Warning: There were {0}  central directory headers found  for  file name {1} `